### PR TITLE
ETNA-794 Add minimal examples to all TSDatasets methods #67

### LINKED ISSUE #67: Add minimal examples to all TSDatasets methods
### 🚀 Feature Request

Add Example sections to all public methods in TSDataset class.

### Motivation

Example section in documentation makes library usage easier.

### Proposal

Add minimal examples to all methods

### Test cases

Run doctest.
Switch on doctest in pytest by default.

### Alternatives

_No response_

### Additional context

_No response_

### Checklist

- [X] Added feature request
- [X] Added motivation
- [X] Added proposal

### PR DESCRIPTION
**Pull request by martins0n**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [x] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna-ts/blob/master/CHANGELOG.md)?

## Type of Change

- [x] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Proposed Changes
Tested examples with doctest in important public methods.

## Closing issues
closes #67

### CODE REVIEW

File: etna/datasets/tsdataset.py [hunk 1]
```diff
@@ -279,7 +365,50 @@ def to_pandas(self, flatten=False) -> pd.DataFrame:
     def to_dataset(df: pd.DataFrame) -> pd.DataFrame:
         """Convert pandas dataframe to ETNA Dataset format.
 
-        Expects pd.DataFrame with columns ["timestamp", "segment"]. Other columns considered features.
+        Parameters
+        ----------
+            df:
```
Comment (Mr-Geekman) [inline etna/datasets/tsdataset.py:370]: Redundant tab.

File: etna/datasets/tsdataset.py [hunk 2]
```diff
@@ -140,6 +160,24 @@ def make_future(self, future_steps: int) -> "TSDataset":
...
+        --------
+        >>> from etna.datasets import generate_const_df
+        >>> df = generate_const_df(
+        ...    periods=30, start_time="2021-06-01",
+        ...    n_segments=2, scale=1
+        ... )
+        >>> df_ts_format = TSDataset.to_dataset(df)
+        >>> ts = TSDataset(df_ts_format, "D")
+        >>> ts.make_future(4)
+        segment    segment_0 segment_1
```
Comment (julia-shenshina) [inline etna/datasets/tsdataset.py:174]: could you please add example with regressors?

Comment (martins0n) [reply etna/datasets/tsdataset.py]: I've added regressors to current example

File: etna/datasets/tsdataset.py (hunk 1 above)
Comment (martins0n) [reply etna/datasets/tsdataset.py]: fix

File: etna/loggers/__init__.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 --------
 >>> from etna.loggers import tslogger, ConsoleLogger
 >>> tslogger.add(ConsoleLogger())
+0
```
Comment (julia-shenshina) [inline etna/loggers/__init__.py:7]:
```suggestion

```
?

**Review by julia-shenshina — DISMISSED** [review]
🔥

File: etna/loggers/__init__.py (hunk 3 above)
Comment (martins0n) [reply etna/loggers/__init__.py:7]: tslogger returns index of added logger.

**Review by julia-shenshina — APPROVED** [review]

**Pull request merged**